mod-source-record-manager-server/src/main/resources/rules/marc_bib_rules.json

```diff
@@ -5653,328 +5653,352 @@
   ],
   "600": [
     {
-      "target": "subjects",
-      "description": "Subject Headings",
-      "subfield": [
-        "a",
-        "b",
-        "c",
-        "d",
-        "e",
-        "f",
-        "g",
-        "h",
-        "j",
-        "k",
-        "l",
-        "m",
-        "n",
-        "o",
-        "p",
-        "q",
-        "r",
-        "s",
-        "t",
-        "u",
-        "v",
-        "x",
-        "y",
-        "z"
-      ],
-      "applyRulesOnConcatenatedData": true,
-      "rules": [
-        {
-          "conditions": [
-            {
-              "type": "trim_period, trim"
-            }
-          ]
-        }
-      ],
-      "subFieldDelimiter": [
+      "entity": [
         {
-          "value": " ",
-          "subfields": [
+          "target": "subjects.value",
+          "description": "Subject Headings",
+          "applyRulesOnConcatenatedData": true,
+          "subfield": [
             "a",
             "b",
             "c",
             "d",
-            "j",
-            "q",
-            "t",
+            "e",
             "f",
             "g",
+            "h",
+            "j",
             "k",
             "l",
             "m",
             "n",
             "o",
             "p",
+            "q",
             "r",
-            "s"
-          ]
-        },
-        {
-          "value": "--",
-          "subfields": [
+            "s",
+            "t",
+            "u",
+            "v",
             "x",
             "y",
-            "z",
-            "v"
+            "z"
+          ],
+          "rules": [
+            {
+              "conditions": [
+                {
+                  "type": "trim_period, trim"
+                }
+              ]
+            }
+          ],
+          "subFieldDelimiter": [
+            {
+              "value": " ",
+              "subfields": [
+                "a",
+                "b",
+                "c",
+                "d",
+                "j",
+                "q",
+                "t",
+                "f",
+                "g",
+                "k",
+                "l",
+                "m",
+                "n",
+                "o",
+                "p",
+                "r",
+                "s"
+              ]
+            },
+            {
+              "value": "--",
+              "subfields": [
+                "x",
+                "y",
+                "z",
+                "v"
+              ]
+            },
+            {
+              "value": "--",
+              "subfields": []
+            }
           ]
         },
         {
-          "value": "--",
-          "subfields": []
+          "target": "series.authorityId",
```

Comment:
Hello, @psmagin, should it be subjects. authorityId?

Reply:
Thanks. Fixed